src/DOM.hs

```diff
@@ -40,8 +40,8 @@ getElementById = ffi "document['getElementById'](%1)"
 getElementsByName :: Text -> Fay [Element]
 getElementsByName = ffi "document['getElementsByName'](%1)"
 
-addEvent :: Text -> Fay f -> Fay ()
-addEvent = ffi "window['addEventListener'](%1,%2)"
+addEvent :: Element -> Text -> Fay f -> Fay ()
+addEvent = ffi "%1['addEventListener'](%1,%2)"
```

Comment:
That's what I get for using the github editor to make changes in place! Thanks, fixed!